licenses.d2iq.yaml

```diff
@@ -503,10 +504,10 @@ resources:
       - license_path: LICENSE
         ref: v${image_tag}
         url: https://github.com/weaviate/weaviate
-  - container_image: ghcr.io/kube-logging/fluentd:v1.17-5.0-full-build.173
+  - container_image: ghcr.io/kube-logging/logging-operator/fluentd:5.2.0-full
     sources:
       - url: https://github.com/fluent/fluentd
```

Comment:
Repo is different - https://github.com/kube-logging/logging-operator/tree/master/images/fluentd

See https://kube-logging.dev/docs/image-versions/